examples/simple/simple_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"log"
+	"testing"
+	"time"
+
+	"github.com/hatchet-dev/hatchet/internal/testutils"
+)
+
+func TestSimpleRun(t *testing.T) {
+	testutils.Prepare(t)
+
+	ch := make(chan interface{}, 1)
+
+	go func() {
+		time.Sleep(5 * time.Second)
+		ch <- struct{}{}
+		log.Printf("sent interrupt")
+	}()
+
+	if err := run(ch); err != nil {
+		t.Fatalf("run() error = %v", err)
+	}
+
+	// TODO test here that the workflow ran by checking the database
```

Comment:
This is obviously missing and important, but I'm not sure about the best way to do it right now